src/db.py

```diff
@@ -99,7 +100,7 @@ def get_job(self, job: JobId) -> JobSchema:
             total_files=int(data.get("total_files", 0)),
             files_errored=int(data.get("files_errored", 0)),
             iterator=data.get("iterator", None),
-            taint=data.get("taint", None),
+            taint=json.loads(data.get("taint", None)),
```

Comment:
change key name in json to `taints`

also this code doesn't make sense right now, because json.loads(None) is an error - change to "[]"